node-fs-readfile/read-any.js

```diff
@@ -0,0 +1,13 @@
+import { readFile } from 'node:fs/promises';
+
+async function readAnyFile() {
+  try {
+    const filePath = process.argv[2];
+    const fileText = await (await readFile(filePath)).toString();
```

Comment:
Why the double `await`?
And why the `.toString()`?
Oh, I get it. You didn't pass an encoding to `readFile` so it returned a Buffer, which you are converting to a string. This wouldn't be necessary if you just pass 'utf8' as the encoding since in that case `readFile` returns a string.

It's great that you got some experience using Buffers, though.